src/views/Analysis/Overview/AnalysisOverview.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { AppState, FinishedGameState } from '../../../state/AppState';
+import { Typography } from '@rmwc/typography';
+import { ListDivider } from '@rmwc/list';
+import styles from './AnalysisOverview.module.css';
+import appRoutes from '../../../routes/routes';
+import { Link } from 'react-router-dom';
+import { GameSummaryCard } from '../../../components/Analysis/GameSummaryCard/GameSummaryCard';
+
+interface AnalysisOverviewProps {
+  finishedGames: FinishedGameState[];
+}
+
+function getLink(game: FinishedGameState) {
+  return {
+    tag: Link,
+    to: appRoutes.AnalysisParam.replace(
+      ':param',
+      game.id ? game.id : game.date,
+    ),
+  };
+}
+
+export function AnalysisOverview(props: AnalysisOverviewProps) {
+  const { finishedGames } = props;
+
+  return (
+    <div className={styles.analysisOverviewLayout}>
+      <div className={styles.header}>
+        <Typography use="headline2" tag="h1">
+          Recently played games
+        </Typography>
+      </div>
+      <ListDivider />
+      <div className={styles.gameList}>
+        {finishedGames.map((game, index) => (
```

Comment:
We should probably only render a small subset of all the games. And then add more games to the tree when the user scrolled down. This could be helpful: https://cassetterocks.github.io/react-infinite-scroller/

Already with 100 games it takes a little while to load. (you can quickly generate games now with an action)

Reply:
Not sure what the best way would be to handle this.

Reply:
I would take paging of the games into an other PR since the is quite some logic involved (I don't like huge PR's). And even more logic when you connect the API to it. For a "normal" user it takes a while to reach this number of game.

Reply:
Okay! I will concentrate on testing this and improving the layout on mobile then so that we can soon merge this.
(the API has no involved in this btw and I also don't plan it to have as of right now)

Reply:
Sorry for making this so huge.
I didn't know that this would require a whole refactor of how we do the board layout..